build

```diff
@@ -32,9 +32,9 @@ export GO15VENDOREXPERIMENT=1
 
 use_go_install=
 
-go_root_dir=$(go env GOROOT)
-go_host_os=$(go env GOHOSTOS)
-go_host_arch=$(go env GOHOSTARCH)
+go_root_dir=$(go env GO_ROOT)
+go_host_os=$(go env GO_HOST_OS)
+go_host_arch=$(go env GO_HOST_ARCH)
```

Comment:
These will break the below `cgodisabled_pkg_dir` variable since `go env GO_ROOT` will return an empty string (GO_ROOT and the other are not valid go env variables).
I think there's no reason to change this.